### PR TITLE
DNS token doc updates

### PR DESCRIPTION
**Pull request by johnlanda**
Review feedback

Update website/content/commands/acl/set-agent-token.mdx

Co-authored-by: Jared Kirschner <85913323+jkirschner-hashicorp@users.noreply.github.com>

Update dns token doc to link to policy templates

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] appropriate backport labels added
* [ ] not a security concern

### CODE REVIEW

**Comment by johnlanda** [comment]
@trujillo-adam if you could take a look at this I would appreciate it! It should be the same as the other you already looked at.

File: website/content/docs/security/acl/tokens/index.mdx [hunk 1]
```diff
@@ -156,6 +156,9 @@ system or accessing Consul under specific conditions. The following table descri
...
+| [`acl.tokens.replication`](/consul/docs/agent/config/config-files#acl_tokens_replication)               | `OPTIONAL` | `N/A`      | Used to authorize secondary datacenters with the primary datacenter for replication operations.                                                                                                                                                   |
+| [`acl.tokens.config_file_service_registration`](/consul/docs/agent/config/config-files#acl_tokens_config_file_service_registration) | `OPTIONAL` | `OPTIONAL` | Used to register services and checks from service and check definitions specified in configuration files or fragments passed to the agent using the `-hcl` flag.                                                      |
+| [`acl.tokens.dns`](/consul/docs/agent/config/config-files#acl_tokens_dns)                               | `OPTIONAL` | `OPTIONAL` | Used by the agent to answer DNS queries about services and nodes.                                                                                                                                                                                 |
```
Comment (trujillo-adam) [inline website/content/docs/security/acl/tokens/index.mdx:161]:
```suggestion
| [`acl.tokens.dns`](/consul/docs/agent/config/config-files#acl_tokens_dns)                               | `OPTIONAL` | `OPTIONAL` | Enables the agent to answer DNS queries about services and nodes.                                                                                                                                                                                 |
```

**Review by trujillo-adam — COMMENTED** [review]
Nice work on this PR - very complete. I added several suggestions for style guide consistency.

**Review by trujillo-adam — APPROVED** [review]
LGTM

**Pull request merged**